### ISSUE TITLE
Does 'distributed tensorflow' support Keras training?

### ISSUE THREAD

**Issue by rayjang** [tensorflow/tensorflow#14692]
[distributed tensorflow in the official site](https://www.tensorflow.org/deploy/distributed)

Instread of tensorflow model,  is Keras model and training possible?

 If I put the Keras model and training, is it working??
![image](https://user-images.githubusercontent.com/9244296/32982719-3394322e-cccc-11e7-9b58-ea5f326e63ff.png)
I want to know if distributed tensorflow support keras model too.

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue? Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue? Please update the label and/or status accordingly.

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
Nagging Awaiting Response: It has been 14 days with no activityand the `awaiting response` label was assigned. Is this still an issue?

**Comment by tensorflowbutler** [comment]
It has been 14 days with no activity and the `awaiting response` label was assigned. Is this still an issue?

**Comment by asimshankar** [comment]
`tf.keras.Model.fit` does not support distributed yet, if that is what you're asking.
However, you can convert a Keras model to an `Estimator` using [`tf.keras.estimator.model_to_estimator`](https://www.tensorflow.org/programmers_guide/estimators#creating_estimators_from_keras_models) and use that for distributed training.

See also: https://www.tensorflow.org/api_docs/python/tf/estimator/train_and_evaluate and https://cloud.google.com/blog/big-data/2018/02/easy-distributed-training-with-tensorflow-using-tfestimatortrain-and-evaluate-on-cloud-ml-engine

Hope that helps.

**Comment by NKUCodingCat** [comment]
@asimshankar Is there any plan to make tf.keras.Model.fit support distribution training? Or users should keep using tf.keras.estimator.model_to_estimator
